cernopendata_client/cli.py

```diff
@@ -40,20 +30,26 @@ def cernopendata_client():
 @cernopendata_client.command()
 @click.option("--recid", type=click.INT, help="Record ID")
 @click.option("--doi", help="Digital Object Identifier.")
-@click.option("--title", help="Record title")
+@click.option("--title", help="Record Title")
 @click.option(
     "--output-fields",
     is_flag=False,
     type=click.STRING,
     help="Comma separated list of fields from the record "
     "that should be included in the output.",
 )
-def get_record(recid, doi, title, output_fields):
+@click.option(
+    "--server",
+    default="http://opendata.cern.ch",
+    type=click.STRING,
+    help="Query Server URL",
+)
+def get_record(recid, doi, title, output_fields, server):
     """Get records content by its recid, doi or title filtered."""
     # TODO: Add decorator to require one of (recid, doi or title)
     if recid is not None:
         validate_recid(recid)
-    record_json = get_record_as_json(recid, doi, title)
+    record_json = get_record_as_json(recid, doi, title, server)
```

Comment:
Please use `server` as the first parameter to all functions.  It seems more intuitive to write `get_value(server, key)` i.e. first saying from which server we are fetching information, and only then saying the inputs.